client/cmdhfmf.c

```diff
@@ -331,11 +331,17 @@ int CmdHF14AMfDump(const char *Cmd)
 	PrintAndLog("|-----------------------------------------|");
 	uint8_t tries = 0;
 	for (sectorNo = 0; sectorNo < numSectors; sectorNo++) {
-		for (tries = 0; tries < 3; tries++) {
-			UsbCommand c = {CMD_MIFARE_READBL, {FirstBlockOfSector(sectorNo) + NumBlocksPerSector(sectorNo) - 1, 0, 0}};
-			memcpy(c.d.asBytes, keyA[sectorNo], 6);
-			SendCommand(&c);
-
+		bool doesKeyAWork = true;

```

Comment:
This is not necessary, The sector trailer can either be read by Key A or not at all. There is no access condition which allows reading by Key B only.